### DISCUSSION TITLE
为什么在 v2 版本中去除 title 的配置

### DISCUSSION THREAD

**Discussion by Mrgao1203** [antvis/G2Plot#3574] [Q&A] [answered]
我还可以通过什么方式来设置 title 呢

**Comment by hizyt** [comment]
我也正在查找当前问题

**Comment by hustcc** [comment, marked as answer]
因为之前的版本，对于标题的样式渲染不够灵活，达不到需求，所以建议大家外置用 div 实现。

不过在[新版本 G2 5.0](https://g2.antv.antgroup.com/spec/component/title) 中解决了这个问题，所以有加回来了。
